praetorian_cli/ui/aegis/commands/job.py

```
@@ -158,20 +187,24 @@ def run_job(menu, args):
+        jobs = menu.sdk.jobs.add(target_key, [capability], config_json, credentials=credentials)
         
         if jobs:
             job = jobs[0] if isinstance(jobs, list) else jobs
             job_key = job.get('key', '')
             status = job.get('status', 'unknown')
-            job_id = job_key.split('#')[-1][:12] if job_key else 'unknown'
-            
-            menu.console.print(f"\n[green]✓ Job {job_id} queued successfully[/green]")
+            job_id = job_key.split('#')[-1] if job_key else 'unknown'
+
+            menu.console.print(f"\n[green]✓ Job queued successfully[/green]")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)